example/README.md

```diff
@@ -0,0 +1,21 @@
+# Garden Tailwind CSS Example
+
+This is a mock implementation of a Zendesk-styled product using
+[Tailwind CSS](https://tailwindcss.com/) and the
+[@zendeskgarden/tailwindcss](https://github.com/zendeskgarden/tailwindcss)
+plugin. It includes no custom styling, with all styles being provided
+by Garden design tokens.
+
+This is not a fully functioning prototype. It is a showcase for different
+technologies and techniques to apply Garden design tokens with Tailwind.
+
+## Technologies
+
+- [create-react-app](https://create-react-app.dev/) with Typescript as the starter project
```

Comment:
Nit: Should this be PascalCase like how the official site has it, [TypeScript](https://github.com/microsoft/TypeScript/wiki)?

Reply:
Updated.